src/main/java/com/google/api/codegen/discogapic/transformer/java/JavaDiscoGapicSchemaToViewTransformer.java

```diff
@@ -168,7 +167,7 @@ private StaticLangApiMessageView generateSchemaClasses(
     // Getters and setters use unescaped name for better readability on public methods.
     schemaView.fieldGetFunction(context.getDiscoGapicNamer().getResourceGetterName(schemaId));
     schemaView.fieldSetFunction(context.getDiscoGapicNamer().getResourceSetterName(schemaId));
-    String schemaTypeName = schemaTypeTable.getAndSaveNicknameForElementType(schema);
+    String schemaTypeName = schemaTypeTable.getAndSaveNicknameFor(schema, true);
```

Comment:
The boolean parameter is unclear at the calling site, i.e. I can't tell what `true` means here. Consider using an enum (since you get names that way).

I see that the function here was private and is now becoming public; it was fine for the parameter to be a boolean when private, because of the smaller scope (it's easier to find the definition and find what it means).